styleguide/_themes/derek/scss/globalElements/basement.scss

```diff
@@ -7,14 +7,19 @@
 
 // Basement Classes
 .basement {
-  padding: 10px 15px;
+  @include make-container;
+  padding: 10px 0;
 }
 
 .basement-copyright {
   color: $gray-base;
   text-align: right;
 }
 
+.basement-row {
+  @extend %row;
```

Comment:
To reiterate what I said elsewhere, I think for consistency purposes it makes sense to just do `@include make-row;` here instead, since that's all that extension is doing anyway.